fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -1391,8 +1447,12 @@ ACTOR static Future<Void> transactionStarter(
 				if (i == 0) { 
 					addActor.send(timeReply(readVersionReply, replyTimes));
 				}
+				nonBatchGRVProcessed += systemTransactionsStarted[i] + defaultPriTransactionsStarted[i];
+				batchGRVProcessed += batchPriTransactionsStarted[i];
 			}
 		}
+		stats->percentageOfGRVQueueProcessed = (double)nonBatchGRVProcessed / (systemQueueSize + defaultQueueSize);
```

Comment:
I think this might be more easily interpreted as a percentage of default priority only. When combined together, it will skew the results (usually slightly) towards a higher percentage, but generally we're more interested in what's happening specifically with default priority since we know all system priority transactions will be processed every time.

Reply:
Done.

Reply:
Ping @sfc-gh-ajbeamon